### ISSUE TITLE
[Feature] No easy way to find wheels

### ISSUE THREAD

**Issue by Please-Pick-a-Name** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#495]
### Is your feature request related to a problem (if not, explain why you think this feature should be added)? Please describe.

There is still no easy way to find the vehicle wheels
certainly not with GetVehicleBodies(), despite the body representation of wheels is certainly a vehicle body

### Describe the solution you'd like

Many
- (easy) `GetVehicleBodies` returns body representation of wheels aswell, **could choose not to if theres other way to get wheels**
- `GetVehicleWheels` return all wheel belongs to the vehicle
- `GetWheelVehicle` return the vehicle that the wheel belongs to
- `GetBodyChildWheels` return all wheels belongs to the body (why not just GetVehicleWheels? semi truck have 2 bodies with wheels, cab and trailer, one may wish to tell it apart)
- `GetWheelParentBody` return the body that the wheel follows
- `GetWheelBody` return the body representation of wheel
- `GetBodyWheel` return the wheel representation of body

### Describe alternatives you've considered

the generic alternative is by loop through all bodies in scene and find body that is static and is a vehicle body, that body then is a wheel body, then one can figure out the handle of the wheel with wheel shape handle and finding the offset between wheel and wheel shape

the other alternative is by using the vehicle handle as base and go check the next fixed amount of handles to find all potential wheels that it have, a better thing at first glance but may well not work for vehicle that used up more handle then considered

the worse alternative is by tagging the wheel shape, then its possible to find the wheel with offset, its however nolonger generic

### Additional context

we are given SetProperties with ability to mess with wheels but no real way of finding wheels:sob:

**Comment by nooitaf** [comment]
A wheel actually has a `wheel` type.. i do this to find wheels in the hover mod.. but this is option 2 you mentioned
```lua
function GetWheelsForVehicle(vehicle)
	local maxHandles = 600 -- maximum handles to check after vehicle handle
	local wheels={}
	for i=vehicle,vehicle + maxHandles do
		if GetEntityType(i) == 'wheel' then
			local shapeBody = GetShapeBody(i+1)
			if GetBodyVehicle(shapeBody) == vehicle then
				table.insert(wheels,i+1)
			else
				break
			end
		end
	end
	return wheels
end
```

**Comment by Gregory-Gregory** [comment]
@Please-Pick-a-Name hi!

Thank you for this suggestion, it has been added to the wishlist for the team's future consideration.

**Comment by YuLun-bili** [comment]
I think this can be marked as resolved as 1.5.4 introduced `GetEntityChildren()` and `GetEntityParent()`
